src/form/textarea/Textarea.tsx

```diff
@@ -0,0 +1,153 @@
+import "./_textarea.scss";
+
+import React, {useState} from "react";
+import classNames from "classnames";
+import TextareaAutosize, {TextareaAutosizeProps} from "react-textarea-autosize";
+
+import {KEYBOARD_EVENT_KEY} from "./util/textareaConstants";
```

Comment:
I think we can use `KEYBOARD_EVENT_KEY` from `"../../core/utils/keyboardEventConstants"` here.

Reply:
You're right, I've missed this. Thank you!